<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="42" total="73" passed="31">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="1275307" started-at="2020-12-03T06:09:50Z" finished-at="2020-12-03T06:31:05Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="1275307" started-at="2020-12-03T06:09:50Z" finished-at="2020-12-03T06:31:05Z">
      <class name="twc.Automation.SmokeTestCases.smokeTestCases">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="beforeClass" is-config="true" duration-ms="155864" started-at="2020-12-03T11:39:50Z" finished-at="2020-12-03T11:42:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="Smoke_Test_CaseVerify_Homescreen_marquee_adCall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_CaseVerify_Homescreen_marquee_adCall_FTL" duration-ms="782307" started-at="2020-12-03T11:42:25Z" finished-at="2020-12-03T11:55:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_CaseVerify_Homescreen_marquee_adCall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aaxSlot_feed1_adCall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aaxSlot_feed1_adCall" duration-ms="434" started-at="2020-12-03T11:55:28Z" finished-at="2020-12-03T11:55:28Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID f4b66249-b6eb-4155-9d90-1e2b04487c99 is not trigred for aax call db_display/feed/feed_1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID f4b66249-b6eb-4155-9d90-1e2b04487c99 is not trigred for aax call db_display/feed/feed_1
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_feed1(Functions.java:6871)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aaxSlot_feed1_adCall(smokeTestCases.java:989)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aaxSlot_feed1_adCall -->
        <test-method status="PASS" signature="Smoke_Test_Verifying_homescreenhourly__adCall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_homescreenhourly__adCall_FTL" duration-ms="384" started-at="2020-12-03T11:55:28Z" finished-at="2020-12-03T11:55:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_homescreenhourly__adCall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aaxSlot_feed2_adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aaxSlot_feed2_adcall" duration-ms="272" started-at="2020-12-03T11:55:29Z" finished-at="2020-12-03T11:55:29Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 752a96eb-3198-4991-b572-17ec04883b6c is not trigred for aax call db_display/feed/feed_2]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 752a96eb-3198-4991-b572-17ec04883b6c is not trigred for aax call db_display/feed/feed_2
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_feed2(Functions.java:6904)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aaxSlot_feed2_adcall(smokeTestCases.java:1002)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aaxSlot_feed2_adcall -->
        <test-method status="PASS" signature="Smoke_Test_verifying_feed_1adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_verifying_feed_1adcall_FTL" duration-ms="318" started-at="2020-12-03T11:55:29Z" finished-at="2020-12-03T11:55:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_verifying_feed_1adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aaxSlot_feed3_adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aaxSlot_feed3_adcall" duration-ms="189" started-at="2020-12-03T11:55:29Z" finished-at="2020-12-03T11:55:30Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 9384272f-b27f-4686-935f-02e6c5763abd is not trigred for aax call db_display/feed/feed_3]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 9384272f-b27f-4686-935f-02e6c5763abd is not trigred for aax call db_display/feed/feed_3
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_feed3(Functions.java:6922)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aaxSlot_feed3_adcall(smokeTestCases.java:1013)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aaxSlot_feed3_adcall -->
        <test-method status="PASS" signature="Smoke_Test_verifying_feed_2adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_verifying_feed_2adcall_FTL" duration-ms="228" started-at="2020-12-03T11:55:30Z" finished-at="2020-12-03T11:55:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_verifying_feed_2adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aaxSlot_feed4_adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aaxSlot_feed4_adcall" duration-ms="188" started-at="2020-12-03T11:55:30Z" finished-at="2020-12-03T11:55:30Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotId 90251553-fb54-47a1-bbe2-dce4e1c27758 is not trigred for aax call db_display/feed/feed_4]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotId 90251553-fb54-47a1-bbe2-dce4e1c27758 is not trigred for aax call db_display/feed/feed_4
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_feed4(Functions.java:6938)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aaxSlot_feed4_adcall(smokeTestCases.java:1025)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aaxSlot_feed4_adcall -->
        <test-method status="PASS" signature="Smoke_Test_verifying_feed_3adCall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_verifying_feed_3adCall_FTL" duration-ms="216" started-at="2020-12-03T11:55:30Z" finished-at="2020-12-03T11:55:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_verifying_feed_3adCall_FTL -->
        <test-method status="PASS" signature="Smoke_Test_verifying_feed_4adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_verifying_feed_4adcall_FTL" duration-ms="152" started-at="2020-12-03T11:55:30Z" finished-at="2020-12-03T11:55:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_verifying_feed_4adcall_FTL -->
        <test-method status="PASS" signature="Smoke_Test_verifying_feed_5adCall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_verifying_feed_5adCall_FTL" duration-ms="164" started-at="2020-12-03T11:55:30Z" finished-at="2020-12-03T11:55:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_verifying_feed_5adCall_FTL -->
        <test-method status="PASS" signature="Smoke_Test_Verify_daily_feedcard_adCall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verify_daily_feedcard_adCall_FTL" duration-ms="158" started-at="2020-12-03T11:55:30Z" finished-at="2020-12-03T11:55:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verify_daily_feedcard_adCall_FTL -->
        <test-method status="PASS" signature="Smoke_Test_Case_Verify_Radar_Maps_deatailspage_adCall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Case_Verify_Radar_Maps_deatailspage_adCall_FTL" duration-ms="206" started-at="2020-12-03T11:55:31Z" finished-at="2020-12-03T11:55:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Case_Verify_Radar_Maps_deatailspage_adCall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_mapsdetails_adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_mapsdetails_adcall" duration-ms="186" started-at="2020-12-03T11:55:31Z" finished-at="2020-12-03T11:55:31Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotId 5db1161b-b504-4640-9496-dfe6284f84ab is not trigred for aax call db_display/details/maps]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotId 5db1161b-b504-4640-9496-dfe6284f84ab is not trigred for aax call db_display/details/maps
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_map_details(Functions.java:3768)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_mapsdetails_adcall(smokeTestCases.java:1058)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_mapsdetails_adcall -->
        <test-method status="PASS" signature="Smoke_Test_CaseVerify_Today_details_page_iu_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_CaseVerify_Today_details_page_iu_FTL" duration-ms="187" started-at="2020-12-03T11:55:31Z" finished-at="2020-12-03T11:55:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_CaseVerify_Today_details_page_iu_FTL -->
        <test-method status="PASS" signature="Smoke_Test_Verify_Air_Quality_contentpage_adCall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verify_Air_Quality_contentpage_adCall_FTL" duration-ms="151" started-at="2020-12-03T11:55:31Z" finished-at="2020-12-03T11:55:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verify_Air_Quality_contentpage_adCall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_today_deatailcard()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_today_deatailcard" duration-ms="242" started-at="2020-12-03T11:55:31Z" finished-at="2020-12-03T11:55:32Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotId 8d4e513d-9ae1-4b32-9468-9be0f434262f  is not trigred for db_display/details/today]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotId 8d4e513d-9ae1-4b32-9468-9be0f434262f  is not trigred for db_display/details/today
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_today_details(Functions.java:3802)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_today_deatailcard(smokeTestCases.java:1092)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_today_deatailcard -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Airquality_deatailcard()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Airquality_deatailcard" duration-ms="193" started-at="2020-12-03T11:55:32Z" finished-at="2020-12-03T11:55:32Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotId 9aec4232-00c6-476b-8bbf-e66ecbd57edb is not trigred for aax call db_display/content/aq]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotId 9aec4232-00c6-476b-8bbf-e66ecbd57edb is not trigred for aax call db_display/content/aq
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_aq(Functions.java:3959)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Airquality_deatailcard(smokeTestCases.java:1104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Airquality_deatailcard -->
        <test-method status="PASS" signature="Smoke_Test_Verify_health_and_activities_Allergy_contenPage_adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verify_health_and_activities_Allergy_contenPage_adcall_FTL" duration-ms="281" started-at="2020-12-03T11:55:32Z" finished-at="2020-12-03T11:55:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verify_health_and_activities_Allergy_contenPage_adcall_FTL -->
        <test-method status="PASS" signature="Smoke_Test_Verify_outdoor_detailpage_adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verify_outdoor_detailpage_adcall_FTL" duration-ms="169" started-at="2020-12-03T11:55:32Z" finished-at="2020-12-03T11:55:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verify_outdoor_detailpage_adcall_FTL -->
        <test-method status="PASS" signature="Smoke_Test_Verify_hourly_detailpage_adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verify_hourly_detailpage_adcall_FTL" duration-ms="27337" started-at="2020-12-03T11:55:32Z" finished-at="2020-12-03T11:56:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verify_hourly_detailpage_adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Hourly_details_adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Hourly_details_adcall" duration-ms="18" started-at="2020-12-03T11:56:00Z" finished-at="2020-12-03T11:56:00Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID ::  9be28769-4207-4d51-8063-dc8e645383b2 is not trigred for aax call db_display/details/hourly]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID ::  9be28769-4207-4d51-8063-dc8e645383b2 is not trigred for aax call db_display/details/hourly
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Hourly(Functions.java:3665)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Hourly_details_adcall(smokeTestCases.java:715)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Hourly_details_adcall -->
        <test-method status="PASS" signature="Smoke_Test_Verify_hourly1_detailpage_bigad_adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verify_hourly1_detailpage_bigad_adcall_FTL" duration-ms="14" started-at="2020-12-03T11:56:00Z" finished-at="2020-12-03T11:56:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verify_hourly1_detailpage_bigad_adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Hourly1_bigaddetails_adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Hourly1_bigaddetails_adcall" duration-ms="12" started-at="2020-12-03T11:56:00Z" finished-at="2020-12-03T11:56:00Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID :: 08f0ccea-cab5-449c-963d-dc57ed9ee87dis not trigred for aax call db_display/details/hourly1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID :: 08f0ccea-cab5-449c-963d-dc57ed9ee87dis not trigred for aax call db_display/details/hourly1
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Hourly1(Functions.java:3683)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Hourly1_bigaddetails_adcall(smokeTestCases.java:726)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Hourly1_bigaddetails_adcall -->
        <test-method status="PASS" signature="Smoke_Test_Verify_hourly2_detailpage_bigad_adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verify_hourly2_detailpage_bigad_adcall_FTL" duration-ms="15" started-at="2020-12-03T11:56:00Z" finished-at="2020-12-03T11:56:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verify_hourly2_detailpage_bigad_adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Hourly2_bigaddetails_adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Hourly2_bigaddetails_adcall" duration-ms="13" started-at="2020-12-03T11:56:00Z" finished-at="2020-12-03T11:56:00Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID :: 4fbed16a-cc6f-4cb1-94f7-81465acbd47edis not trigred for aax call db_display/details/hourly2]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID :: 4fbed16a-cc6f-4cb1-94f7-81465acbd47edis not trigred for aax call db_display/details/hourly2
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Hourly2(Functions.java:3701)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Hourly2_bigaddetails_adcall(smokeTestCases.java:737)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Hourly2_bigaddetails_adcall -->
        <test-method status="FAIL" signature="Smoke_Test_Verify_hourly3__bigad_detailpage_adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verify_hourly3__bigad_detailpage_adcall_FTL" duration-ms="14" started-at="2020-12-03T11:56:00Z" finished-at="2020-12-03T11:56:00Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[/7646/app_android_us/db_display/details/hourly3 call was not trigred]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: /7646/app_android_us/db_display/details/hourly3 call was not trigred
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.Verify_hourly_detailpage_bigad3_adcall_iu(Functions.java:2818)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_Verify_hourly3__bigad_detailpage_adcall_FTL(smokeTestCases.java:315)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verify_hourly3__bigad_detailpage_adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Hourly3_bigaddetails_adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Hourly3_bigaddetails_adcall" duration-ms="13" started-at="2020-12-03T11:56:00Z" finished-at="2020-12-03T11:56:00Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID :: 2634dc9-b59f-4b2c-b281-bb3be291b7b6 is not trigred for aax call db_display/details/hourly3]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID :: 2634dc9-b59f-4b2c-b281-bb3be291b7b6 is not trigred for aax call db_display/details/hourly3
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Hourly3(Functions.java:3720)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Hourly3_bigaddetails_adcall(smokeTestCases.java:749)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Hourly3_bigaddetails_adcall -->
        <test-method status="PASS" signature="Smoke_Test_Verifying_newdailydetails_day1adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_newdailydetails_day1adcall_FTL" duration-ms="24419" started-at="2020-12-03T11:56:00Z" finished-at="2020-12-03T11:56:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_newdailydetails_day1adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Daily_details_Day1_adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Daily_details_Day1_adcall" duration-ms="179" started-at="2020-12-03T11:56:24Z" finished-at="2020-12-03T11:56:24Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_thu1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_thu1
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Daily(Functions.java:3752)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Daily_details_Day1_adcall(smokeTestCases.java:767)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Daily_details_Day1_adcall -->
        <test-method status="PASS" signature="Smoke_Test_Verifying_newdailydetails_day2adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_newdailydetails_day2adcall_FTL" duration-ms="9901" started-at="2020-12-03T11:56:24Z" finished-at="2020-12-03T11:56:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_newdailydetails_day2adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Dailydetails_Day2adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Dailydetails_Day2adcall" duration-ms="108" started-at="2020-12-03T11:56:34Z" finished-at="2020-12-03T11:56:34Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_fri1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_fri1
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Daily(Functions.java:3752)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Dailydetails_Day2adcall(smokeTestCases.java:778)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Dailydetails_Day2adcall -->
        <test-method status="FAIL" signature="Smoke_Test_Verifying_newdailydetails_day3adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_newdailydetails_day3adcall_FTL" duration-ms="9709" started-at="2020-12-03T11:56:34Z" finished-at="2020-12-03T11:56:44Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[daily details ad call was not trigred]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: daily details ad call was not trigred
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.verifyingdailydetailiu(Functions.java:5783)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_Verifying_newdailydetails_day3adcall_FTL(smokeTestCases.java:363)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_newdailydetails_day3adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Dailydetails_Day3adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Dailydetails_Day3adcall" duration-ms="217" started-at="2020-12-03T11:56:44Z" finished-at="2020-12-03T11:56:44Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_sat1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_sat1
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Daily(Functions.java:3752)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Dailydetails_Day3adcall(smokeTestCases.java:790)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Dailydetails_Day3adcall -->
        <test-method status="PASS" signature="Smoke_Test_Verifying_newdailydetails_day4adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_newdailydetails_day4adcall_FTL" duration-ms="10077" started-at="2020-12-03T11:56:44Z" finished-at="2020-12-03T11:56:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_newdailydetails_day4adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Dailydetails_Day4adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Dailydetails_Day4adcall" duration-ms="137" started-at="2020-12-03T11:56:54Z" finished-at="2020-12-03T11:56:55Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_sun1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_sun1
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Daily(Functions.java:3752)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Dailydetails_Day4adcall(smokeTestCases.java:802)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Dailydetails_Day4adcall -->
        <test-method status="PASS" signature="Smoke_Test_Verifying_newdailydetails_day5adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_newdailydetails_day5adcall_FTL" duration-ms="9320" started-at="2020-12-03T11:56:55Z" finished-at="2020-12-03T11:57:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_newdailydetails_day5adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Dailydetails_Day5adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Dailydetails_Day5adcall" duration-ms="211" started-at="2020-12-03T11:57:04Z" finished-at="2020-12-03T11:57:04Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_mon1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_mon1
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Daily(Functions.java:3752)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Dailydetails_Day5adcall(smokeTestCases.java:812)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Dailydetails_Day5adcall -->
        <test-method status="PASS" signature="Smoke_Test_Verifying_newdailydetails_day6adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_newdailydetails_day6adcall_FTL" duration-ms="9600" started-at="2020-12-03T11:57:04Z" finished-at="2020-12-03T11:57:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_newdailydetails_day6adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Dailydetails_Day6adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Dailydetails_Day6adcall" duration-ms="168" started-at="2020-12-03T11:57:14Z" finished-at="2020-12-03T11:57:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_tue1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_tue1
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Daily(Functions.java:3752)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Dailydetails_Day6adcall(smokeTestCases.java:825)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Dailydetails_Day6adcall -->
        <test-method status="FAIL" signature="Smoke_Test_Verifying_newdailydetails_day7adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_newdailydetails_day7adcall_FTL" duration-ms="9841" started-at="2020-12-03T11:57:14Z" finished-at="2020-12-03T11:57:24Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null is not matched with wed1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null is not matched with wed1
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.verifyingdailydetailiu(Functions.java:5804)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_Verifying_newdailydetails_day7adcall_FTL(smokeTestCases.java:420)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_newdailydetails_day7adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Dailydetails_Day7adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Dailydetails_Day7adcall" duration-ms="133" started-at="2020-12-03T11:57:24Z" finished-at="2020-12-03T11:57:24Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_wed1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_wed1
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Daily(Functions.java:3752)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Dailydetails_Day7adcall(smokeTestCases.java:837)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Dailydetails_Day7adcall -->
        <test-method status="PASS" signature="Smoke_Test_Verifying_newdailydetails_day8adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_newdailydetails_day8adcall_FTL" duration-ms="12784" started-at="2020-12-03T11:57:24Z" finished-at="2020-12-03T11:57:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_newdailydetails_day8adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Dailydetails_Day8adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Dailydetails_Day8adcall" duration-ms="148" started-at="2020-12-03T11:57:37Z" finished-at="2020-12-03T11:57:37Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_thu2]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_thu2
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Daily(Functions.java:3752)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Dailydetails_Day8adcall(smokeTestCases.java:849)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Dailydetails_Day8adcall -->
        <test-method status="PASS" signature="Smoke_Test_Verifying_newdailydetails_day9adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_newdailydetails_day9adcall_FTL" duration-ms="12682" started-at="2020-12-03T11:57:37Z" finished-at="2020-12-03T11:57:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_newdailydetails_day9adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Dailydetails_Day9adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Dailydetails_Day9adcall" duration-ms="85" started-at="2020-12-03T11:57:49Z" finished-at="2020-12-03T11:57:50Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_fri2]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_fri2
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Daily(Functions.java:3752)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Dailydetails_Day9adcall(smokeTestCases.java:863)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Dailydetails_Day9adcall -->
        <test-method status="PASS" signature="Smoke_Test_Verifying_newdailydetails_day10adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_newdailydetails_day10adcall_FTL" duration-ms="12706" started-at="2020-12-03T11:57:50Z" finished-at="2020-12-03T11:58:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_newdailydetails_day10adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Dailydetails_Day10adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Dailydetails_Day10adcall" duration-ms="133" started-at="2020-12-03T11:58:02Z" finished-at="2020-12-03T11:58:02Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_sat2]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_sat2
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Daily(Functions.java:3752)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Dailydetails_Day10adcall(smokeTestCases.java:876)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Dailydetails_Day10adcall -->
        <test-method status="PASS" signature="Smoke_Test_Verifying_newdailydetails_day11adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_newdailydetails_day11adcall_FTL" duration-ms="12684" started-at="2020-12-03T11:58:02Z" finished-at="2020-12-03T11:58:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_newdailydetails_day11adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Dailydetails_Day11adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Dailydetails_Day11adcall" duration-ms="235" started-at="2020-12-03T11:58:15Z" finished-at="2020-12-03T11:58:15Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_sun2]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_sun2
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Daily(Functions.java:3752)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Dailydetails_Day11adcall(smokeTestCases.java:889)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Dailydetails_Day11adcall -->
        <test-method status="PASS" signature="Smoke_Test_Verifying_newdailydetails_day12adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_newdailydetails_day12adcall_FTL" duration-ms="12605" started-at="2020-12-03T11:58:15Z" finished-at="2020-12-03T11:58:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_newdailydetails_day12adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Dailydetails_Day12adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Dailydetails_Day12adcall" duration-ms="595" started-at="2020-12-03T11:58:28Z" finished-at="2020-12-03T11:58:28Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_mon2]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_mon2
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Daily(Functions.java:3752)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Dailydetails_Day12adcall(smokeTestCases.java:900)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Dailydetails_Day12adcall -->
        <test-method status="PASS" signature="Smoke_Test_Verifying_newdailydetails_day13adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_newdailydetails_day13adcall_FTL" duration-ms="12721" started-at="2020-12-03T11:58:28Z" finished-at="2020-12-03T11:58:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_newdailydetails_day13adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Dailydetails_Day13adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Dailydetails_Day13adcall" duration-ms="112" started-at="2020-12-03T11:58:41Z" finished-at="2020-12-03T11:58:41Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_tue2]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_tue2
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Daily(Functions.java:3752)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Dailydetails_Day13adcall(smokeTestCases.java:913)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Dailydetails_Day13adcall -->
        <test-method status="PASS" signature="Smoke_Test_Verifying_newdailydetails_day14adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_newdailydetails_day14adcall_FTL" duration-ms="12820" started-at="2020-12-03T11:58:41Z" finished-at="2020-12-03T11:58:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_newdailydetails_day14adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Dailydetails_Day14adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Dailydetails_Day14adcall" duration-ms="236" started-at="2020-12-03T11:58:54Z" finished-at="2020-12-03T11:58:54Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_wed2]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_wed2
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Daily(Functions.java:3752)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Dailydetails_Day14adcall(smokeTestCases.java:926)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Dailydetails_Day14adcall -->
        <test-method status="FAIL" signature="Smoke_Test_Verifying_newdailydetails_day15adcall_FTL()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_newdailydetails_day15adcall_FTL" duration-ms="12964" started-at="2020-12-03T11:58:54Z" finished-at="2020-12-03T11:59:07Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[daily details ad call was not trigred]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: daily details ad call was not trigred
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.verifyingdailydetailsiu2(Functions.java:5915)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_Verifying_newdailydetails_day15adcall_FTL(smokeTestCases.java:534)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_newdailydetails_day15adcall_FTL -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Dailydetails_Day15adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Dailydetails_Day15adcall" duration-ms="116" started-at="2020-12-03T11:59:07Z" finished-at="2020-12-03T11:59:07Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_thu3]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for db_display/details/10day_thu3
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Daily(Functions.java:3752)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Dailydetails_Day15adcall(smokeTestCases.java:940)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Dailydetails_Day15adcall -->
        <test-method status="PASS" signature="Smoke_Test_CaseVerify_WeatherFXAPI_url_KillLaunch()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_CaseVerify_WeatherFXAPI_url_KillLaunch" duration-ms="65979" started-at="2020-12-03T11:59:07Z" finished-at="2020-12-03T12:00:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_CaseVerify_WeatherFXAPI_url_KillLaunch -->
        <test-method status="PASS" signature="Smoke_Test_CaseVerify_Factualapi_url_KillLaunch()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_CaseVerify_Factualapi_url_KillLaunch" duration-ms="134" started-at="2020-12-03T12:00:13Z" finished-at="2020-12-03T12:00:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_CaseVerify_Factualapi_url_KillLaunch -->
        <test-method status="PASS" signature="Smoke_Test_CaseVerify_turbo_url_KillLaunch()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_CaseVerify_turbo_url_KillLaunch" duration-ms="92" started-at="2020-12-03T12:00:14Z" finished-at="2020-12-03T12:00:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_CaseVerify_turbo_url_KillLaunch -->
        <test-method status="FAIL" signature="Smoke_Test__Verifying_Bcp_apicall_URL_killLaunch()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test__Verifying_Bcp_apicall_URL_killLaunch" duration-ms="65" started-at="2020-12-03T12:00:14Z" finished-at="2020-12-03T12:00:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[https://bcp.crwdcntrl.net/ url was not trigred]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: https://bcp.crwdcntrl.net/ url was not trigred
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.validating_bcp_privacy_Optinmode_scenarion(Functions.java:4310)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test__Verifying_Bcp_apicall_URL_killLaunch(smokeTestCases.java:589)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test__Verifying_Bcp_apicall_URL_killLaunch -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Hourly_preroll_adcall_killLaunch()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Hourly_preroll_adcall_killLaunch" duration-ms="59" started-at="2020-12-03T12:00:14Z" finished-at="2020-12-03T12:00:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID ::  9be28769-4207-4d51-8063-dc8e645383b2 is not trigred for aax call db_display/details/hourly]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID ::  9be28769-4207-4d51-8063-dc8e645383b2 is not trigred for aax call db_display/details/hourly
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Hourly(Functions.java:3665)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Hourly_preroll_adcall_killLaunch(smokeTestCases.java:606)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Hourly_preroll_adcall_killLaunch -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Hourly1_bigaddetails_preroll_adcall_killLaunch()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Hourly1_bigaddetails_preroll_adcall_killLaunch" duration-ms="61" started-at="2020-12-03T12:00:14Z" finished-at="2020-12-03T12:00:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID :: 08f0ccea-cab5-449c-963d-dc57ed9ee87dis not trigred for aax call db_display/details/hourly1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID :: 08f0ccea-cab5-449c-963d-dc57ed9ee87dis not trigred for aax call db_display/details/hourly1
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Hourly1(Functions.java:3683)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Hourly1_bigaddetails_preroll_adcall_killLaunch(smokeTestCases.java:617)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Hourly1_bigaddetails_preroll_adcall_killLaunch -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Hourly2_bigaddetails_preroll_adcall_killLaunch()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Hourly2_bigaddetails_preroll_adcall_killLaunch" duration-ms="55" started-at="2020-12-03T12:00:14Z" finished-at="2020-12-03T12:00:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID :: 4fbed16a-cc6f-4cb1-94f7-81465acbd47edis not trigred for aax call db_display/details/hourly2]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID :: 4fbed16a-cc6f-4cb1-94f7-81465acbd47edis not trigred for aax call db_display/details/hourly2
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Hourly2(Functions.java:3701)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Hourly2_bigaddetails_preroll_adcall_killLaunch(smokeTestCases.java:629)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Hourly2_bigaddetails_preroll_adcall_killLaunch -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Hourly3_bigaddetails_preroll_adcall_killLaunch()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Hourly3_bigaddetails_preroll_adcall_killLaunch" duration-ms="58" started-at="2020-12-03T12:00:14Z" finished-at="2020-12-03T12:00:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID :: 2634dc9-b59f-4b2c-b281-bb3be291b7b6 is not trigred for aax call db_display/details/hourly3]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID :: 2634dc9-b59f-4b2c-b281-bb3be291b7b6 is not trigred for aax call db_display/details/hourly3
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Hourly3(Functions.java:3720)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Hourly3_bigaddetails_preroll_adcall_killLaunch(smokeTestCases.java:641)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Hourly3_bigaddetails_preroll_adcall_killLaunch -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_mapsdetails_preload_adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_mapsdetails_preload_adcall" duration-ms="60" started-at="2020-12-03T12:00:14Z" finished-at="2020-12-03T12:00:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotId 5db1161b-b504-4640-9496-dfe6284f84ab is not trigred for aax call db_display/details/maps]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotId 5db1161b-b504-4640-9496-dfe6284f84ab is not trigred for aax call db_display/details/maps
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_map_details(Functions.java:3768)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_mapsdetails_preload_adcall(smokeTestCases.java:654)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_mapsdetails_preload_adcall -->
        <test-method status="FAIL" signature="Smoke_Test_verify_amazon_Slotid_homescreenhourly_preroll_adcall_killLaunch()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_verify_amazon_Slotid_homescreenhourly_preroll_adcall_killLaunch" duration-ms="63" started-at="2020-12-03T12:00:14Z" finished-at="2020-12-03T12:00:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 869c843c-7cf8-47ae-b6ed-088057e4bc8a is not trigred for aax call db_display/home_screen/hourly]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 869c843c-7cf8-47ae-b6ed-088057e4bc8a is not trigred for aax call db_display/home_screen/hourly
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_homescreen_hourly(Functions.java:3616)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_verify_amazon_Slotid_homescreenhourly_preroll_adcall_killLaunch(smokeTestCases.java:665)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_verify_amazon_Slotid_homescreenhourly_preroll_adcall_killLaunch -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aaxSlot_feed1_preroladcall_killLaunch()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aaxSlot_feed1_preroladcall_killLaunch" duration-ms="69" started-at="2020-12-03T12:00:14Z" finished-at="2020-12-03T12:00:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID f4b66249-b6eb-4155-9d90-1e2b04487c99 is not trigred for aax call db_display/feed/feed_1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID f4b66249-b6eb-4155-9d90-1e2b04487c99 is not trigred for aax call db_display/feed/feed_1
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_feed1(Functions.java:6871)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aaxSlot_feed1_preroladcall_killLaunch(smokeTestCases.java:676)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aaxSlot_feed1_preroladcall_killLaunch -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aaxSlot_feed2_preroladcall_KillLaunch()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aaxSlot_feed2_preroladcall_KillLaunch" duration-ms="59" started-at="2020-12-03T12:00:14Z" finished-at="2020-12-03T12:00:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 752a96eb-3198-4991-b572-17ec04883b6c is not trigred for aax call db_display/feed/feed_2]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 752a96eb-3198-4991-b572-17ec04883b6c is not trigred for aax call db_display/feed/feed_2
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_feed2(Functions.java:6904)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aaxSlot_feed2_preroladcall_KillLaunch(smokeTestCases.java:689)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aaxSlot_feed2_preroladcall_KillLaunch -->
        <test-method status="FAIL" signature="Smoke_Test_amazon_aax_Daily_details_preroll_adcall()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_amazon_aax_Daily_details_preroll_adcall" duration-ms="61" started-at="2020-12-03T12:00:14Z" finished-at="2020-12-03T12:00:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for daily details]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: slotID 6c5a145d-9198-48f4-adfd-08f05557eace is not trigred for daily details
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.get_aaxcal_Daily1(Functions.java:6888)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_amazon_aax_Daily_details_preroll_adcall(smokeTestCases.java:702)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_amazon_aax_Daily_details_preroll_adcall -->
        <test-method status="FAIL" signature="Smoke_Test_Verifying_homescreenhourly_pulltoreresh_scenario_killLaunch()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_homescreenhourly_pulltoreresh_scenario_killLaunch" duration-ms="50473" started-at="2020-12-03T12:00:14Z" finished-at="2020-12-03T12:01:05Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[/7646/app_android_us/db_display/home_screen/hourly call was not trigred]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: /7646/app_android_us/db_display/home_screen/hourly call was not trigred
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.finding_Homescreen_iu_value(Functions.java:2339)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_Verifying_homescreenhourly_pulltoreresh_scenario_killLaunch(smokeTestCases.java:963)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_homescreenhourly_pulltoreresh_scenario_killLaunch -->
        <test-method status="FAIL" signature="Smoke_Test_Verifying_homescreenmarquee__pulltoreresh_scenario()[pri:0, instance:twc.Automation.SmokeTestCases.smokeTestCases@1dcad16f]" name="Smoke_Test_Verifying_homescreenmarquee__pulltoreresh_scenario" duration-ms="10" started-at="2020-12-03T12:01:05Z" finished-at="2020-12-03T12:01:05Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[/7646/app_android_us/db_display/home_screen/marquee call was not trigred]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: /7646/app_android_us/db_display/home_screen/marquee call was not trigred
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.General.Functions.finding_Homescreen_marquee_iu_value(Functions.java:4045)
	at twc.Automation.SmokeTestCases.smokeTestCases.Smoke_Test_Verifying_homescreenmarquee__pulltoreresh_scenario(smokeTestCases.java:974)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Smoke_Test_Verifying_homescreenmarquee__pulltoreresh_scenario -->
      </class> <!-- twc.Automation.SmokeTestCases.smokeTestCases -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
